ansible/roles/gateone-gen-sshkey/tasks/main.yml

```diff
@@ -30,5 +29,30 @@
   delegate_to: gateone
 
 - name: finally, transfer the public key locally
-  fetch: src="/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh/{{ ATMOUSERNAME }}_default.pub" dest="{{ GATEONE_LOCAL_PUBKEY_DIR | default('/tmp/') }}" flat=yes
+  fetch: src= "/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh/{{ ATMOUSERNAME }}_default.pub" dest= "{{ GATEONE_LOCAL_PUBKEY_DIR | default('/tmp/') }}" flat=yes
   delegate_to: gateone
+
+# THESE values were in atmo-user-ssh-key and throw an error at thi stage.
+# TODO: discuss if this is required..
+
+# - name: grab local copy of user ssh private key
+#   set_fact: pk_file= "{{ lookup('file', '/tmp/' + ATMOUSERNAME + '_default.pub') }}"
+#   tags: debug
+# 
+# - name: distribute the generated gate one key to the instance
+#   authorized_key: >
+#     key= '{{ pk_file }}'
+#     state=present
+#     user= "{{ item }}"
+#   when: pk_file is defined
+#   with_items:
+#     - "{{ ATMOUSERNAME }}"
+#   tags:
+#     - ssh-keys
+#     - debug
+# 
+# - name: remove user key from controller node
+#   local_action: file path=/tmp/{{ ATMOUSERNAME }}_default.pub state=absent
+#   tags:
+#     - ssh-keys
+#     - delete-key
```

Comment:
This role expects the previous role to be run before-hand, since that role is responsible for generating and depositing keys on the instance for GateOne.  To fix this, please run `gateone-gen-ssh-key`.  This is the section of code that is responsible for that file: https://github.com/iPlantCollaborativeOpenSource/atmosphere-ansible/blob/master/ansible/roles/gateone-gen-sshkey/tasks/main.yml#L32-L34